src/status_im/contexts/preview/quo/main.cljs

```diff
@@ -243,7 +244,9 @@
                         :component collection-avatar/view}
                        {:name      :account-avatar
                         :component account-avatar/view}]
-   :banner            [{:name      :banner
+   :banner            [{:name      :alert-banners
```

Comment:
I tried `:alert-banner` as the name but the preview wasn't working properly. So I changed it to `:alert-banners`